src/main/java/org/mule/tools/api/repository/RepositoryGenerator.java

```diff
@@ -44,14 +42,11 @@
   protected File outputDirectory;
 
   public RepositoryGenerator(MavenProject project, List<ArtifactRepository> remoteArtifactRepositories, File outputDirectory,
```

Comment:
If this class requires a maven logger it can not be part of the API. Please revisit this remove the log or move the class back to where it was